pkg/login/social/models/models.go

```diff
@@ -0,0 +1,76 @@
+package models
```

Comment:
I had to move OAuthInfo to models to prevent cyclic dependencies between social service and sso settings service.

Reply:
Yep because you embed `ssosettings.Reloadable` in `SocialConnector` interface. Instead you can just define the required functions directly and you wont have a cyclic dependency